MsBox.Avalonia/Controls/MsBoxCustomView.axaml.cs

```diff
@@ -29,8 +29,11 @@ public string GetButtonResult()
     public Task Copy()
     {
         var clipboard = TopLevel.GetTopLevel(this).Clipboard;
-        var vm = this.DataContext as MsBoxCustomViewModel;

```

Comment:
this was always copying the entire content, not the selected part. Now we are only copying the entire text if nothing is selected